Wow:

```
  ../../../../../../worker/src/DepLibUring.cpp: In constructor ‘DepLibUring::LibUring::LibUring()’:
  ../../../../../../worker/src/DepLibUring.cpp:316:30: error: invalid pure specifier (only ‘= 0’ is allowed) before ‘err’
    316 |                 int errno = -err;
        |                              ^~~
  In file included from ../../../../../../worker/subprojects/libuv-v1.47.0/include/uv/errno.h:25,
                   from ../../../../../../worker/subprojects/libuv-v1.47.0/include/uv.h:56,
                   from ../../../../../../worker/include/DepLibUV.hpp:5,
                   from ../../../../../../worker/include/DepLibUring.hpp:4,
                   from ../../../../../../worker/src/DepLibUring.cpp:4:
  ../../../../../../worker/src/DepLibUring.cpp:316:21: error: function ‘int* __errno_location()’ is initialized like a variable
    316 |                 int errno = -err;
        |                     ^~~~~
  In file included from ../../../../../../worker/src/DepLibUring.cpp:5:
  ../../../../../../worker/include/Logger.hpp:216:101: warning: too many arguments for format [-Wformat-extra-args]
    216 |                         const int loggerWritten = std::snprintf(Logger::buffer, Logger::bufferSize, "W" _MS_LOG_STR_DESC desc, _MS_LOG_ARG, ##__VA_ARGS__); \
  ../../../../../../worker/src/DepLibUring.cpp:322:25: note: in expansion of macro ‘MS_WARN_TAG’
    322 |                         MS_WARN_TAG(
        |                         ^~~~~~~~~~~
```

Also wiht this change:

```
./../../src/DepLibUring.cpp:316:30: error: invalid pure specifier (only ‘= 0’ is allowed) before numeric constant
  316 |                 int errno = -1 * err;
      |                              ^
In file included from ../../../subprojects/libuv-v1.47.0/include/uv/errno.h:25,
                 from ../../../subprojects/libuv-v1.47.0/include/uv.h:56,
                 from ../../../include/DepLibUV.hpp:5,
                 from ../../../include/DepLibUring.hpp:4,
                 from ../../../src/DepLibUring.cpp:4:
../../../src/DepLibUring.cpp:316:21: error: function ‘int* __errno_location()’ is initialized like a variable
  316 |                 int errno = -1 * err;
      |                     ^~~~~
ninja: build stopped: subcommand failed.
```

--- a/worker/src/DepLibUring.cpp
+++ b/worker/src/DepLibUring.cpp
@@ -35,7 +35,7 @@ inline static void onFdEvent(uv_poll_t* handle, int status, int events)
 	if (err < 0)
 	{
 		// Get positive errno.
-		int error = -1 * err;
+		int error = -err;
 
 		MS_ABORT("eventfd_read() failed: %s", std::strerror(error));
 	};
@@ -282,7 +282,7 @@ DepLibUring::LibUring::LibUring()
 	if (err < 0)
 	{
 		// Get positive errno.
-		int error = -1 * err;
+		int error = -err;
 
 		MS_THROW_ERROR("io_uring_queue_init() failed: %s", std::strerror(error));
 	}
@@ -300,7 +300,7 @@ DepLibUring::LibUring::LibUring()
 	if (err < 0)
 	{
 		// Get positive errno.
-		int error = -1 * err;
+		int error = -err;
 
 		MS_THROW_ERROR("io_uring_register_eventfd() failed: %s", std::strerror(error));
 	}
@@ -324,7 +324,7 @@ DepLibUring::LibUring::LibUring()
 	if (err < 0)
 	{
 		// Get positive errno.
-		int error = -1 * err;
+		int error = -err;
 
 		if (error == ENOMEM)
 		{
@@ -352,7 +352,7 @@ DepLibUring::LibUring::~LibUring()
 	if (err != 0)
 	{
 		// Get positive errno.
-		int error = -1 * err;
+		int error = -err;
 
 		MS_ABORT("close() failed: %s", std::strerror(error));
 	}
@@ -575,7 +575,7 @@ void DepLibUring::LibUring::Submit()
 	else
 	{
 		// Get positive errno.
-		int error = -1 * err;
+		int error = -err;
 
 		MS_ERROR("io_uring_submit() failed: %s", std::strerror(error));
 	}

--- a/worker/src/DepLibUring.cpp
+++ b/worker/src/DepLibUring.cpp
@@ -313,20 +313,21 @@ DepLibUring::LibUring::LibUring()
 
 	if (err < 0)
 	{
-		int errno = -1 * err;
+		// Get positive errno.
+		int error = -1 * err;
 
-		if (errno == ENOMEM)
+		if (error == ENOMEM)
 		{
 			this->zeroCopyEnabled = false;
 
 			MS_WARN_TAG(
 			  info,
 			  "io_uring_register_buffers() failed due to low memlock limit (ulimit -l), disabling zero copy: %s",
-			  std::strerror(errno));
+			  std::strerror(error));
 		}
 		else
 		{
-			MS_THROW_ERROR("io_uring_register_buffers() failed: %s", std::strerror(errno));
+			MS_THROW_ERROR("io_uring_register_buffers() failed: %s", std::strerror(error));
 		}
 	}
 }

--- a/worker/src/DepLibUring.cpp
+++ b/worker/src/DepLibUring.cpp
@@ -31,9 +31,13 @@ inline static void onFdEvent(uv_poll_t* handle, int status, int events)
 	// the counter in order to avoid libuv calling this callback indefinitely.
 	eventfd_t v;
 	int err = eventfd_read(liburing->GetEventFd(), std::addressof(v));
+
 	if (err < 0)
 	{
-		MS_ABORT("eventfd_read() failed: %s", std::strerror(-err));
+		// Get positive errno.
+		int error = -1 * err;
+
+		MS_ABORT("eventfd_read() failed: %s", std::strerror(error));
 	};
 
 	for (unsigned int i{ 0 }; i < count; ++i)
@@ -277,7 +281,10 @@ DepLibUring::LibUring::LibUring()
 
 	if (err < 0)
 	{
-		MS_THROW_ERROR("io_uring_queue_init() failed: %s", std::strerror(-err));
+		// Get positive errno.
+		int error = -1 * err;
+
+		MS_THROW_ERROR("io_uring_queue_init() failed: %s", std::strerror(error));
 	}
 
 	// Create an eventfd instance.
@@ -292,7 +299,10 @@ DepLibUring::LibUring::LibUring()
 
 	if (err < 0)
 	{
-		MS_THROW_ERROR("io_uring_register_eventfd() failed: %s", std::strerror(-err));
+		// Get positive errno.
+		int error = -1 * err;
+
+		MS_THROW_ERROR("io_uring_register_eventfd() failed: %s", std::strerror(error));
 	}
 
 	// Initialize available UserData entries.
@@ -341,7 +351,10 @@ DepLibUring::LibUring::~LibUring()
 
 	if (err != 0)
 	{
-		MS_ABORT("close() failed: %s", std::strerror(-err));
+		// Get positive errno.
+		int error = -1 * err;
+
+		MS_ABORT("close() failed: %s", std::strerror(error));
 	}
 
 	// Close the ring.
@@ -561,7 +574,10 @@ void DepLibUring::LibUring::Submit()
 	}
 	else
 	{
-		MS_ERROR("io_uring_submit() failed: %s", std::strerror(-err));
+		// Get positive errno.
+		int error = -1 * err;
+
+		MS_ERROR("io_uring_submit() failed: %s", std::strerror(error));
 	}
 }
 

